### DISCUSSION TITLE
Authorization Denied: No client certificate was presented

### DISCUSSION THREAD

**Discussion by vishnu-nj** [Keyfactor/ejbca-ce#97] [Q&A] [answered]
Hi, I am new to EJBCA and followed the instructions at https://hub.docker.com/r/primekey/ejbca-ce, under 'Quick start - classic workflow' to run it as a docker image on my windows 11 machine. I installed the SuperAdmin client certificate and tried to access the adminweb and it gave the below error, kindly help

Authorization Denied
No client certificate was presented
If you did not get prompted to select a client certificate, please check that you have the correct certificate.

**Comment by Ib23** [comment, marked as answer]
Please see link here 
- https://github.com/Keyfactor/ejbca-ce/issues/65

**Reply by vishnu-nj** [reply]
Hi, are you aware on how to start the docker container in non production mode ?

**Reply by primetomas** [reply]
non-production mode is compile time setting for some things, so it is not possible to enable it in the container without swapping out the ejbca.ear file.
